docusaurus/docs/Android/02-client/06-guides/03-channel-list-updates.mdx

```diff
@@ -0,0 +1,124 @@
+# Channel List Updates
+
+The `ChannelList` component is being updated based on the data from two sources:
+- Query channels request, with given filter and query sort, when the component is initialized
+- Channel-related events that come from the `WebSocket` when the socket is connected
+
+Unlike query channel requests, where you can filter particular channels, channel-related events are not being filtered and need additional attention to handle the list updates correctly.
+By default, the SDK updates the list based on membership - the channel will be added if the currentUser is a member and will be removed otherwise.
```

Comment:
```suggestion
By default, the SDK updates the list based on membership - the channel will be added if the `currentUser` is a member and will be removed otherwise.
```